CMakeLists.txt

```diff
@@ -573,6 +573,10 @@ configure_file(src/glfw3.pc.in
 # The library is installed by src/CMakeLists.txt
 #--------------------------------------------------------------------
 if (GLFW_INSTALL)
+    if(IS_ABSOLUTE "${GLFW_INSTALL_CONFIG_DIR}")
```

Comment:
This doesn't seem to be necessary - it is not an OPTION so shouldn't be overriden.